docs/Reference.md

```diff
@@ -231,6 +231,9 @@ User-Agent: DiscordBot ($url, $versionNumber)
 
 Clients may append more information and metadata to the _end_ of this string as they wish.
 
+> warn
+> Client requests that do not have a valid User Agent specified may be blocked and lead to a Cloudflare error page.
```

Comment:
this is a small thing, but including "page" seems a little confusing. maybe switch to the language they use on [their docs](https://support.cloudflare.com/hc/en-us/articles/360029779472-Troubleshooting-Cloudflare-1XXX-errors#h_e6ba4204-ab4f-464b-afdc-e8177e418e34)?

could do "page" -> "in the HTML body"

Reply:
In the example of that ticket, the `error code: 1010` is the body of the response rather than a header or the response type (which is still `403`). You can verify by removing the `-D` flag which dumps the header